front/src/views/Gifs.vue

```diff
@@ -16,7 +16,13 @@
         <div class="gifs__container">
             <div class="gifs__container__list">
                 <div class="gifs__container__list__item" v-for="gif in gifs" :key="gif.id">
-                    <img :src="gif.images.fixed_width.url" :alt="gif.title" />
+                    <img
+                        :src="gif.images.fixed_width.url"
+                        :alt="gif.title"
+                        @load="loadComplete(gif.id)"
+                        :hidden="!loadedGifs.includes(gif.id)"
+                    />
+                    <v-skeleton-loader v-if="!loadedGifs.includes(gif.id)" class="mx-auto" type="image"></v-skeleton-loader>
```

Comment:
why a v-if here and a :hidden for <img> ?

Reply:
Because a need to load image to catch @load